Formula/wasmtime.rb

```diff
@@ -0,0 +1,28 @@
+class Wasmtime < Formula
+  desc "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+  homepage "https://wasmtime.dev/"
+  license "Apache-2.0"
+  head "https://github.com/bytecodealliance/wasmtime.git"
+
+  stable do
+    url "https://github.com/bytecodealliance/wasmtime.git", tag: "v0.28.0"
+  end
+
+  livecheck do
+    url :head
+    strategy :github_latest
```

Comment:
Are you sure `:github_latest` is needed here? We usually try to avoid using it, as it entails making API calls that get rate-limited.

Reply:
There are some mess with tags, for example there is a tag `0.46.1` which is two years old and the latest version is `0.28.0`. That is why I used `github_latest` strategy here.

Reply:
Can we use a tighter regex to rule out the old tag instead? We can also use `:stable` instead of `:head` here.

Reply:
Where are many tags, starting from `v0.31.0` and up to `v0.46.1`.

Reply:
Ok, fair enough. CC @samford in case you have thoughts here.

Reply:
The [`wasmtime` repository](https://github.com/bytecodealliance/wasmtime) was forked from the [now archived] [`cranelift` repository](https://github.com/bytecodealliance/cranelift). Unfortunately, this means that the `wasmtime` repo contains all the previous `cranelift` tags (e.g., `v0.46.1`). New `cranelift` tags in the `wasmtime` repo use a format like `cranelift-v0.69.0` but the older tags are indistinguishable from `wasmtime` versions.

As a result, we can't use the `Git` strategy with the standard regex for Git tags like `1.2.3`/`v1.2.3` (`v?(\d+(?:\.\d+)+)`), as the newest version will be incorrectly given as `0.46.1` (an old `cranelift` version) instead of `0.28.0` (the "latest" `wasmtime` version on GitHub). This would only work if upstream removed the old `cranelift` tags from the `wasmtime` repository.

Long story short, the `GithubLatest` strategy is unfortunately necessary here to identify the correct latest version, due to the weirdness in the upstream repository.